src/components/sessions/SessionDetailView/index.ts

```diff
@@ -0,0 +1 @@
+export { ALL_CLASSES_TAB_INDEX, default } from "./SessionDetailView";
```

Comment:
I'm curious, what is this file for? I thought these are being exported from `SessionDetailView` already

Reply:
this makes it so that we can group our files (component + tests) under in subdirectories, and lets us keep import statements short!

without it, we have to do:
```
import ... from "../components/sessions/session-detail-view/SessionDetailView"
```

compared to:
```
import ... from "../components/sessions/session-detail-view"
```

Reply:
gotcha! looks good B)